include/net/wifi_credentials.h

```diff
@@ -192,6 +194,14 @@ typedef void (*wifi_credentials_ssid_cb)(void *cb_arg, const char *ssid, size_t
  */
 void wifi_credentials_for_each_ssid(wifi_credentials_ssid_cb cb, void *cb_arg);
 
+/**
+ * @brief Converts Wi-Fi credentials configured via Kconfigs into Wi-Fi management params.
+ *
+ * @param params       Pointer to struct in which Wi-Fi connect request params are stored
+ */
+int wifi_credentials_load_static(struct wifi_connect_req_params *params);
```

Comment:
offline teams comment: Instead of this API, use `NET_REQUEST_WIFI_CONNECT_STORED` net_mgmt call